internal/handlers/tigris/msg_getparameter.go

```diff
@@ -17,10 +17,87 @@ package tigris
 import (
 	"context"
 
+	"github.com/FerretDB/FerretDB/internal/handlers/common"
+	"github.com/FerretDB/FerretDB/internal/types"
+	"github.com/FerretDB/FerretDB/internal/util/lazyerrors"
+	"github.com/FerretDB/FerretDB/internal/util/must"
 	"github.com/FerretDB/FerretDB/internal/wire"
 )
 
 // MsgGetParameter implements HandlerInterface.
 func (h *Handler) MsgGetParameter(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error) {
-	return nil, errNotImplemented
+	document, err := msg.Document()
+	if err != nil {
+		return nil, lazyerrors.Error(err)
+	}
+
+	command := document.Command()
+
+	getParameter, err := document.Get(command)
+	if err != nil {
+		return nil, lazyerrors.Error(err)
+	}
+
+	resDB := must.NotFail(types.NewDocument(
+		"acceptApiVersion2", false,
+		"authSchemaVersion", int32(5),
+		"quiet", false,
+		"ok", float64(1),
+	))
+
+	var reply wire.OpMsg
+	resDoc := resDB
+	if getParameter != "*" {
+		resDoc, err = selectParam(document, resDB)
+		if err != nil {
+			return nil, lazyerrors.Error(err)
+		}
+	}
+
+	err = reply.SetSections(wire.OpMsgSection{Documents: []*types.Document{resDoc}})
+	if err != nil {
+		return nil, lazyerrors.Error(err)
+	}
+
+	common.Ignored(document, h.L, "comment")
+
+	if resDoc.Len() < 2 {
+		return &reply, common.NewErrorMsg(common.ErrorCode(0), "no option found to get")
+	}
+
+	return &reply, nil
+}
+
+// selectParam is makes a selection of requested parameters.
```

Comment:
Definitely not `is` and `makes` together :)
Maybe something like this?

```suggestion
// selectParam returns a document consists of keys requested in `document` and values presented in `resDoc`.
```

A better wording is better, but I guess you got my idea.